src/NServiceBus.RabbitMQ/Configuration/DelayedDeliverySettings.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using Configuration.AdvanceExtensibility;
+    using Settings;
+
+    /// <summary>
+    /// TODO: Fill this out
+    /// </summary>
+    public class DelayedDeliverySettings : ExposeSettings
+    {
+        internal DelayedDeliverySettings(SettingsHolder settings) : base(settings) { }
+
+        /// <summary>
+        /// TODO: Fill this out
+        /// </summary>
+        public void DisableTimeoutManager()
```

Comment:
@SeanFeldman I agree with the order, but what I don't like is writing configuration in 3 lines when It can be done in only 1. In the examples that I send there is no ordering enforced. I am not saying that FluentAPI is what we need. But now we make the configuration a bit cumbersome